core/src/banking_stage.rs

```diff
@@ -380,10 +381,14 @@ impl BankingStage {
 
     fn reset_cost_tracker_if_new_bank(
         cost_tracker: &Arc<RwLock<CostTracker>>,
-        bank_slot: Slot,
+        bank: Arc<Bank>,
         banking_stage_stats: &BankingStageStats,
     ) {
-        cost_tracker.write().unwrap().reset_if_new_bank(bank_slot);
+        if !bank.feature_set.is_active(&feature_set::cost_model::id()) {
```

Comment:
assuming bank inherits feature_set from its ancestors, that it is safe to fetch feature_set from current bank rather than from `root_bank`. Can someone comment if this is the preferred practice?